apollo-server/v3/integration-testing/src/__tests__/server.test.ts

```diff
@@ -0,0 +1,36 @@
+// we import a function that we wrote to create a new instance of Apollo Server
+import { createApolloServer } from '../server';
+
+// we will use supertest to test our server
+import request from 'supertest';
+
+// this is the query we use for our test
+const queryData = {
```

Comment:
In an even more ideal world we use `graphql-code-generator` to end up with strong types for the return value from this query but that can be part of a later change to get codegen to be a part of the default AS experience :)